FreeRTOS/FreeRTOS-Kernel
# Code: Fix stream buffer warning

Code: Fix stream buffer warning

Description
-----------
Remove unnecessary assert condition 

Test Steps
-----------
Compilation no longer gives a warning
```
Overall coverage rate:
  lines......: 96.2% (1233 of 1282 lines)
  functions..: 100.0% (91 of 91 functions)
  branches...: 93.9% (682 of 726 branches)
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
